packages/ui/src/common/hooks/useRefetchQueries.ts

```diff
@@ -25,15 +26,19 @@ export const useRefetchQueries = (
       if (interval) {
         const handler = setInterval(() => {
           apolloClient.refetchQueries({ include })
+          isRefetched.current = true
```

Comment:
`isRefetched` is a good idea for list of entities where the default is `[]` instead of `undefined` but if it starts refetching before the query even completed the UI might show something like "There are no current proposals yet" when it's not the case. To solve this we could:
1. Return `undefined` when `data` is `undefined` but it means changing a bunch of hooks (also I just discovered a [`networkStatus` prop](https://www.apollographql.com/docs/react/data/queries#inspecting-loading-states) that could be used but it might not be necessary) on top of that something would have to be passed to `useRefetched` `when` to not refetch not completed query.
2. I also just found out that `refetchQueries` has a [`onQueryUpdated` option](https://www.apollographql.com/docs/react/data/refetching/#refetching-selectively). The way I understand it the second param can be used refetch only completed query. But even better in the 3rd example the query is updated accordingly to it's cache policy with: `return observableQuery.reobserve()` I think it would make the `isLoading` not switch back to `true` (I'm not sure though...)

Reply:
Fix applied in new commit